lib/src/components/NxThreatIndicator/__tests__/NxThreatIndicator.test.tsx

```diff
@@ -4,116 +4,104 @@
-    expect(getShallowComponent({ policyThreatLevel: 9 }).find('.nx-threat-indicator'))
-        .not.toMatchSelector('.nx-threat-indicator--unspecified');
+    await user.hover(threatIndicator);
+    const tooltip = await screen.findByRole('tooltip');
+    expect(tooltip).toHaveTextContent('Moderate');
   });
 
-  it('sets the modifier class using the threatLevelCategory if both props are provided', function() {
-    const component = getShallowComponent({ policyThreatLevel: 9, threatLevelCategory: 'low' });
-    const icon = component.find(NxFontAwesomeIcon);
+  it('takes precedence of threatLevelCategory if both props are provided', async function() {
+    const { container } = quickRender({ policyThreatLevel: 9, threatLevelCategory: 'low' }),
+        user = userEvent.setup(),
+        threatIndicator = container.querySelector('svg')!;
 
-    expect(icon).toMatchSelector('.nx-threat-indicator--low');
-    expect(icon).not.toMatchSelector('.nx-threat-indicator--critical');
-    expect(icon).not.toMatchSelector('.nx-threat-indicator--unspecified');
+    await user.hover(threatIndicator);
+    const tooltip = await screen.findByRole('tooltip');
+    expect(tooltip).toHaveTextContent('Low');
   });
 
   it('adds aria attrs to help the icon show up for screen readers', function() {
-    const component = getShallowComponent({ policyThreatLevel: 9, threatLevelCategory: 'low' });
-
-    expect(component.find(NxFontAwesomeIcon)).toHaveProp('aria-label', 'threat level low');
-    expect(component.find(NxFontAwesomeIcon)).not.toHaveProp('aria-hidden');
-  });
+    const { container } = quickRender({ policyThreatLevel: 9, threatLevelCategory: 'low' }),
+        threatIndicator = container.querySelector('svg')!;
 
-  it('should have the correct default tooltip title', function() {
-    const none = getShallowComponent({ threatLevelCategory: 'none' }).find(NxToolip);
-    const low = getShallowComponent({ threatLevelCategory: 'low' }).find(NxToolip);
-    const moderate = getShallowComponent({ threatLevelCategory: 'moderate' }).find(NxToolip);
-    const severe = getShallowComponent({ threatLevelCategory: 'severe' }).find(NxToolip);
-    const critical = getShallowComponent({ threatLevelCategory: 'critical' }).find(NxToolip);
-
-    expect(none).toHaveProp('title', 'None');
-    expect(low).toHaveProp('title', 'Low');
-    expect(moderate).toHaveProp('title', 'Moderate');
-    expect(severe).toHaveProp('title', 'Severe');
-    expect(critical).toHaveProp('title', 'Critical');
+    expect(threatIndicator).toHaveAttribute('aria-label', 'threat level low');
+    expect(threatIndicator).not.toHaveAttribute('aria-hidden', true);
   });
 
-  it('should show custom tooltip title', function() {
-    const tooltip = getShallowComponent({ title: 'Extinction Level Threat' }).find(NxToolip);
-
-    expect(tooltip).toHaveProp('title', 'Extinction Level Threat');
+  describe('should have default tooltips', function() {
```

Comment:
> Since there's a finite and small-ish number of possibilities here (11 threat levels numbers, six threat level categories) I think we should test all of them to make sure they have the correct tooltip text.

I agree. Added tests for all threat numbers here: f7f94af8862571762ca4971a06dada5d133d16a1